### DISCUSSION TITLE
New svg icon that actually represents bluesky that's not just a square

### DISCUSSION THREAD

**Discussion by LukaTV939** [bluesky-social/atproto#1754] [Bluesky Lexicons]
I saw [this post](https://bsky.app/profile/gdcolon.com/post/3kc4hzo2xbz2u) on how the Bluesky logo svg is just a square. Why is there no alternative that actually represents this social media in some shape or form?

**Comment by boylett** [comment]
I've mocked up a few icons based on [Bluesky's website logo](https://blueskyweb.xyz/images/logo-64x64.jpg). Personally I like numbers three and four and will be using them in [my icomoon brands font](https://github.com/boylett/Social-Brands-Webfont) ☺️

**Version 1**

![bluesky-1](https://github.com/bluesky-social/atproto/assets/145017198/a8c768e7-ed1a-4bb1-9c8c-cd166d155353)

```svg
<svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
  <defs>
    <linearGradient gradientTransform="translate(-206.462 -206.462) scale(1.40325)" gradientUnits="userSpaceOnUse" id="a" x1="514.273" x2="511.84" y1="177.771" y2="847.458">
      <stop offset="0" stop-color="#0062ff"/>
      <stop offset="1" stop-color="#0190ff"/>
    </linearGradient>
  </defs>
  <path d="M221.616 42C122.417 42 42 122.417 42 221.616v580.768C42 901.583 122.417 982 221.616 982h580.768C901.583 982 982 901.583 982 802.384V221.616C982 122.417 901.583 42 802.384 42H221.616Zm242.016 269.073 83.8 71.784 92.483-70.644 184.527 140.149-182.773-87.747-95.07 80.38-98.754-77.354-133.571 21.575 149.358-78.143ZM220.607 505.247l65.032 50.648 75.687-34.818 83.932 66.041-85.072-25.785-80.906 45.298-63.277-62.137-88.142 7.63 92.746-46.877Zm505.388 85.817s36.548 30.583 36.835 30.959c.288.377 38.107-29.468 38.107-29.468l66.435 39.247-66.04-8.332-34.029 31.793-43.983-29.995-81.3 28.46 83.975-62.664Z" fill="url(#a)"/>
</svg>
```

---

**Version 2**

![bluesky-2](https://github.com/bluesky-social/atproto/assets/145017198/dc992ac2-b54a-4476-9734-c6a2f4401a68)

```svg
<svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
  <path d="M221.616 42C122.417 42 42 122.417 42 221.616v580.768C42 901.583 122.417 982 221.616 982h580.768C901.583 982 982 901.583 982 802.384V221.616C982 122.417 901.583 42 802.384 42H221.616Zm242.016 269.073 83.8 71.784 92.483-70.644 184.527 140.149-182.773-87.747-95.07 80.38-98.754-77.354-133.571 21.575 149.358-78.143ZM220.607 505.247l65.032 50.648 75.687-34.818 83.932 66.041-85.072-25.785-80.906 45.298-63.277-62.137-88.142 7.63 92.746-46.877Zm505.388 85.817s36.548 30.583 36.835 30.959c.288.377 38.107-29.468 38.107-29.468l66.435 39.247-66.04-8.332-34.029 31.793-43.983-29.995-81.3 28.46 83.975-62.664Z"/>
</svg>
```

---

**Version 3**

![bluesky-3](https://github.com/bluesky-social/atproto/assets/145017198/f7aa2d90-06eb-4517-a7a3-1b707db2875e)

```svg
<svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
  <path d="M221.625 42C122.426 42 42 122.426 42 221.625v580.75C42 901.574 122.426 982 221.625 982h580.75C901.574 982 982 901.574 982 802.375v-580.75C982 122.426 901.574 42 802.375 42h-580.75Zm434.969 260.219a4.965 4.965 0 0 1 3.062.812l211.125 140.344c4.953 3.292.919 10.919-4.593 8.687l-200.032-81a4.962 4.962 0 0 0-5.281 1.032L547.469 480.906c-1.688 1.62-4.29 1.855-6.219.532l-131.469-90.25a4.911 4.911 0 0 0-3.875-.75l-142.5 32.312c-5.459 1.238-8.443-6.13-3.656-9.031l161.5-97.875c1.726-1.046 3.919-.964 5.531.25l110.531 83.25a4.956 4.956 0 0 0 6.219-.219l110.157-95.687a4.904 4.904 0 0 1 2.906-1.219Zm-420.25 255.093a3.395 3.395 0 0 1 1.656.907l74.562 73.375a3.413 3.413 0 0 0 3.438.812l100.656-32.844a3.386 3.386 0 0 1 3.438.813l85.812 85.313c2.654 2.637-.319 6.991-3.75 5.5l-89.844-39.063a3.397 3.397 0 0 0-2.656-.031l-109.344 45.562a3.388 3.388 0 0 1-3.937-1l-70.937-87.968c-.621-.77-1.575-1.221-2.563-1.25l-97.687-2.876c-3.722-.109-4.558-5.277-1.063-6.562l110.344-40.562a3.362 3.362 0 0 1 1.875-.126Zm548.594 136.75 82.781 42.75c1.737.898 1.013 3.541-.938 3.407l-78.156-5.375a1.83 1.83 0 0 0-1.437.562l-40.907 42.813a1.78 1.78 0 0 1-2.219.281l-58.218-35.031a1.811 1.811 0 0 0-1.625-.125l-93.344 38.781c-1.903.79-3.423-1.738-1.844-3.063l97-81.343s50.468 37.556 50.875 38.031c.408.475 48.031-41.688 48.032-41.688Z"/>
</svg>
```

---

**Version 4**

![bluesky-4](https://github.com/bluesky-social/atproto/assets/145017198/3f95cb99-3289-457d-9265-73cad7559949)

```svg
<svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
  <path d="M345.292 220.365a6.963 6.963 0 0 1 7.84.396l156.795 118.833a6.966 6.966 0 0 0 8.802-.309L674.99 202.658a6.964 6.964 0 0 1 8.465-.549l299.48 200.36c7.025 4.7 1.318 15.585-6.501 12.399L692.668 299.241a6.964 6.964 0 0 0-7.473 1.447L524.324 456.066a6.965 6.965 0 0 1-8.808.726L329.022 327.943a6.961 6.961 0 0 0-5.51-1.062l-202.117 46.147c-7.744 1.768-11.968-8.753-5.177-12.896l229.074-139.767ZM188.517 502.078a4.65 4.65 0 0 1 4.956.698l110.205 94.142a4.654 4.654 0 0 0 4.798.766l135.328-55.849a4.649 4.649 0 0 1 4.815.781l126.885 109.486c3.924 3.385.294 9.729-4.585 8.012l-127.757-44.957a4.64 4.64 0 0 0-3.652.243l-146.015 74.351a4.652 4.652 0 0 1-5.522-.983L181.285 574.063a4.659 4.659 0 0 0-3.62-1.485l-134.892 6.019c-5.138.23-6.822-6.845-2.138-8.984l147.882-67.535ZM655.21 695.76s75.503 56.576 76.113 57.291c.609.715 71.841-62.793 71.841-62.793l123.847 64.385c2.598 1.351 1.496 5.317-1.422 5.115l-116.918-8.095a2.687 2.687 0 0 0-2.133.832l-61.177 64.458a2.682 2.682 0 0 1-3.337.45l-87.086-52.744a2.68 2.68 0 0 0-2.422-.181l-139.654 58.364c-2.847 1.19-5.126-2.588-2.764-4.582l145.112-122.5Z"/>
</svg>
```

**Comment by LukaTV939** [comment]
> I've mocked up a few icons based on [Bluesky's website logo](https://blueskyweb.xyz/images/logo-64x64.jpg). Personally I like numbers three and four and will be using them in [my icomoon brands font](https://github.com/boylett/Social-Brands-Webfont) ☺️
> 
> **Version 1**
> 
> ![bluesky-1](https://github.com/bluesky-social/atproto/assets/145017198/a8c768e7-ed1a-4bb1-9c8c-cd166d155353)
> 
> ```svg
> <svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
>   <defs>
>     <linearGradient gradientTransform="translate(-206.462 -206.462) scale(1.40325)" gradientUnits="userSpaceOnUse" id="a" x1="514.273" x2="511.84" y1="177.771" y2="847.458">
>       <stop offset="0" stop-color="#0062ff"/>
>       <stop offset="1" stop-color="#0190ff"/>
>     </linearGradient>
>   </defs>
>   <path d="M221.616 42C122.417 42 42 122.417 42 221.616v580.768C42 901.583 122.417 982 221.616 982h580.768C901.583 982 982 901.583 982 802.384V221.616C982 122.417 901.583 42 802.384 42H221.616Zm242.016 269.073 83.8 71.784 92.483-70.644 184.527 140.149-182.773-87.747-95.07 80.38-98.754-77.354-133.571 21.575 149.358-78.143ZM220.607 505.247l65.032 50.648 75.687-34.818 83.932 66.041-85.072-25.785-80.906 45.298-63.277-62.137-88.142 7.63 92.746-46.877Zm505.388 85.817s36.548 30.583 36.835 30.959c.288.377 38.107-29.468 38.107-29.468l66.435 39.247-66.04-8.332-34.029 31.793-43.983-29.995-81.3 28.46 83.975-62.664Z" fill="url(#a)"/>
> </svg>
> ```
> 
> ---
> 
> **Version 2**
> 
> ![bluesky-2](https://github.com/bluesky-social/atproto/assets/145017198/dc992ac2-b54a-4476-9734-c6a2f4401a68)
> 
> ```svg
> <svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
>   <path d="M221.616 42C122.417 42 42 122.417 42 221.616v580.768C42 901.583 122.417 982 221.616 982h580.768C901.583 982 982 901.583 982 802.384V221.616C982 122.417 901.583 42 802.384 42H221.616Zm242.016 269.073 83.8 71.784 92.483-70.644 184.527 140.149-182.773-87.747-95.07 80.38-98.754-77.354-133.571 21.575 149.358-78.143ZM220.607 505.247l65.032 50.648 75.687-34.818 83.932 66.041-85.072-25.785-80.906 45.298-63.277-62.137-88.142 7.63 92.746-46.877Zm505.388 85.817s36.548 30.583 36.835 30.959c.288.377 38.107-29.468 38.107-29.468l66.435 39.247-66.04-8.332-34.029 31.793-43.983-29.995-81.3 28.46 83.975-62.664Z"/>
> </svg>
> ```
> 
> ---
> 
> **Version 3**
> 
> ![bluesky-3](https://github.com/bluesky-social/atproto/assets/145017198/f7aa2d90-06eb-4517-a7a3-1b707db2875e)
> 
> ```svg
> <svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
>   <path d="M221.625 42C122.426 42 42 122.426 42 221.625v580.75C42 901.574 122.426 982 221.625 982h580.75C901.574 982 982 901.574 982 802.375v-580.75C982 122.426 901.574 42 802.375 42h-580.75Zm434.969 260.219a4.965 4.965 0 0 1 3.062.812l211.125 140.344c4.953 3.292.919 10.919-4.593 8.687l-200.032-81a4.962 4.962 0 0 0-5.281 1.032L547.469 480.906c-1.688 1.62-4.29 1.855-6.219.532l-131.469-90.25a4.911 4.911 0 0 0-3.875-.75l-142.5 32.312c-5.459 1.238-8.443-6.13-3.656-9.031l161.5-97.875c1.726-1.046 3.919-.964 5.531.25l110.531 83.25a4.956 4.956 0 0 0 6.219-.219l110.157-95.687a4.904 4.904 0 0 1 2.906-1.219Zm-420.25 255.093a3.395 3.395 0 0 1 1.656.907l74.562 73.375a3.413 3.413 0 0 0 3.438.812l100.656-32.844a3.386 3.386 0 0 1 3.438.813l85.812 85.313c2.654 2.637-.319 6.991-3.75 5.5l-89.844-39.063a3.397 3.397 0 0 0-2.656-.031l-109.344 45.562a3.388 3.388 0 0 1-3.937-1l-70.937-87.968c-.621-.77-1.575-1.221-2.563-1.25l-97.687-2.876c-3.722-.109-4.558-5.277-1.063-6.562l110.344-40.562a3.362 3.362 0 0 1 1.875-.126Zm548.594 136.75 82.781 42.75c1.737.898 1.013 3.541-.938 3.407l-78.156-5.375a1.83 1.83 0 0 0-1.437.562l-40.907 42.813a1.78 1.78 0 0 1-2.219.281l-58.218-35.031a1.811 1.811 0 0 0-1.625-.125l-93.344 38.781c-1.903.79-3.423-1.738-1.844-3.063l97-81.343s50.468 37.556 50.875 38.031c.408.475 48.031-41.688 48.032-41.688Z"/>
> </svg>
> ```
> 
> ---
> 
> **Version 4**
> 
> ![bluesky-4](https://github.com/bluesky-social/atproto/assets/145017198/3f95cb99-3289-457d-9265-73cad7559949)
> 
> ```svg
> <svg height="1024" stroke-miterlimit="10" width="1024" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
>   <path d="M345.292 220.365a6.963 6.963 0 0 1 7.84.396l156.795 118.833a6.966 6.966 0 0 0 8.802-.309L674.99 202.658a6.964 6.964 0 0 1 8.465-.549l299.48 200.36c7.025 4.7 1.318 15.585-6.501 12.399L692.668 299.241a6.964 6.964 0 0 0-7.473 1.447L524.324 456.066a6.965 6.965 0 0 1-8.808.726L329.022 327.943a6.961 6.961 0 0 0-5.51-1.062l-202.117 46.147c-7.744 1.768-11.968-8.753-5.177-12.896l229.074-139.767ZM188.517 502.078a4.65 4.65 0 0 1 4.956.698l110.205 94.142a4.654 4.654 0 0 0 4.798.766l135.328-55.849a4.649 4.649 0 0 1 4.815.781l126.885 109.486c3.924 3.385.294 9.729-4.585 8.012l-127.757-44.957a4.64 4.64 0 0 0-3.652.243l-146.015 74.351a4.652 4.652 0 0 1-5.522-.983L181.285 574.063a4.659 4.659 0 0 0-3.62-1.485l-134.892 6.019c-5.138.23-6.822-6.845-2.138-8.984l147.882-67.535ZM655.21 695.76s75.503 56.576 76.113 57.291c.609.715 71.841-62.793 71.841-62.793l123.847 64.385c2.598 1.351 1.496 5.317-1.422 5.115l-116.918-8.095a2.687 2.687 0 0 0-2.133.832l-61.177 64.458a2.682 2.682 0 0 1-3.337.45l-87.086-52.744a2.68 2.68 0 0 0-2.422-.181l-139.654 58.364c-2.847 1.19-5.126-2.588-2.764-4.582l145.112-122.5Z"/>
> </svg>
> ```
> 

This actually looks awesome
Gonna use the first one for my littlelink page
Thanks

**Comment by snarfed** [comment]
A number of people tried their hand at this a while ago. The team themselves considered a stylized butterfly at one point: https://bsky.app/profile/birdapp.rip/post/3jt7hpz3jwb2t

<img width=300 src="https://github.com/bluesky-social/atproto/assets/778068/d7be68b0-3e54-4035-8c87-f0891f3c0a53">
<img width=300 src="https://github.com/bluesky-social/atproto/assets/778068/4596b379-90e2-434a-b15c-dde80c1a0f8d">

...and someone from the community iterated on it and flattened it a bit: https://bsky.app/profile/vito.fyi/post/3jt7sst7vok2u

<img width="300" alt="image" src="https://github.com/bluesky-social/atproto/assets/778068/edc94900-6b31-4981-a1c9-eac0ef899776">

They also tried a flattened cloud: https://bsky.app/profile/vito.fyi/post/3jqkauv6krk2y

<img width=300 src="https://github.com/bluesky-social/atproto/assets/778068/351af218-5471-4d41-a299-9d2e35c4eb4a">

I liked all of these! I used the flattened buttefly in https://bluesky-atom.appspot.com/

**Comment by pfrazee** [comment]
We do have a logo on the way, design is finalized, shouldn't be too long now. Sorry for the wait! 🦋

**Reply by haiyankong** [reply]
Hi, here is the bluesky new logo! https://github.com/OzzyCzech/bluesky-icon

**Reply by Diti** [reply]
[It’s a dead butterfly](https://www.emilydamstra.com/please-enough-dead-butterflies/). Any chance to replace it with an alive one?

**Reply by LukaTV939** [reply]
> [It’s a dead butterfly](https://www.emilydamstra.com/please-enough-dead-butterflies/). Any chance to replace it with an alive one?

Lowkey they could've made a good april fools out of that fact
